src/msg/async/rdma/RDMAStack.cc

```diff
@@ -244,17 +244,20 @@ void RDMADispatcher::polling()
       perf_logger->set(l_msgr_rdma_inflight_tx_chunks, inflight);
       if (num_dead_queue_pair) {
         Mutex::Locker l(lock); // FIXME reuse dead qp because creating one qp costs 1 ms
-        for (auto &i : dead_queue_pairs) {
+        auto it = dead_queue_pairs.begin();
```

Comment:
my bad, i didn't realize that we are removing elements while iterating thru the vector.

Reply:
Thanks for your quick review, anyway.